src/index.ts

```diff
@@ -0,0 +1,65 @@
+import es from 'event-stream';
+import type { AggregationCursor, Document, FindCursor } from 'mongodb';
+import type { Stream } from 'stream';
```

Comment:
`Stream` is a very legacy class – it's not even documented anymore – maybe you'd rather want to go with `Readable` here?